libs/design/src/core/theming/types/theme.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * The possible states of a theme.
+ */
+export enum DaffTheme {
+	Light = 'light',
+	Dark = 'dark',
+  None = 'none'
```

Comment:
Indent?

Reply:
It's just weird github formatting, looks fine in file